openassessment/xblock/static/js/src/lms/oa_base.js

```diff
@@ -124,29 +122,32 @@ OpenAssessment.BaseView.prototype = {
 
         // If we don't have anywhere to put the message, just log it to the console
         if (container === null) {
-            if (msg !== null) { console.log(msg); }
+            if (message !== null) { console.log(message); }
         }
 
         else {
             // Insert the error message
-            var msgHtml = (msg === null) ? "" : msg;
+            var msgHtml = (message === null) ? "" : message;
             $(container + " .message__content", element).html('<p>' + msgHtml + '</p>');
             // Toggle the error class
-            $(container, element).toggleClass('has--error', msg !== null);
+            $(container, element).toggleClass('has--error', message !== null);
         }
     },
 
     /**
-    Report an error loading a step.
-
-    Args:
-        step (str): the step that could not be loaded.
-    **/
-    showLoadError: function(step) {
-        var container = '#openassessment__' + step;
-        $(container).toggleClass('has--error', true);
-        $(container + ' .step__status__value i').removeClass().addClass('icon fa fa-exclamation-triangle');
-        $(container + ' .step__status__value .copy').html(gettext('Unable to Load'));
+     * Report an error loading a step.
+     *
+     * @param {string} stepName The step that could not be loaded.
+     * @param {string} errorMessage An optional error message to use instead of the default.
+     */
+    showLoadError: function(stepName, errorMessage) {
+        if (!errorMessage) {
+            errorMessage = gettext('Unable to load');
+        }
+        var $container = $('#openassessment__' + stepName);
+        $container.toggleClass('has--error', true);
+        $container.find('.step__status__value i').removeClass().addClass('icon fa fa-exclamation-triangle');
+        $container.find('.step__status__value .copy').html(errorMessage);
```

Comment:
Strangely there are some callers that pass a message as a second parameter, even though I only introduced that as an option with this change. Given that, I've chosen to just escape the string.